pkg/yaml/yaml.go

```diff
@@ -74,6 +75,15 @@ func subValues(fullMap map[string]interface{}, subMap map[string]interface{}, en
 				if err != nil {
 					return err
 				}
+			case []interface{}:
```

Comment:
The logic change is here. I added a case that handles slices. Previously when a slice came in was just dropped on the floor.